app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,32 @@
+<div id="<%= dom_id recipe %>">
+  <p>
+    <strong>Name:</strong>
+    <%= recipe.name %>
+  </p>
+
+  <p>
+    <strong>Preparation time:</strong>
+    <%= recipe.preparation_time %>
+  </p>
+
+  <p>
+    <strong>Cooking time:</strong>
+    <%= recipe.cooking_time %>
+  </p>
+
```

Comment:
- [ ] You did great work on this project but you still need to work on something for your project to be completed , as we can see on the wireframe provided , we have the specific time for how long the food will be deliverd but I can see that  you foget to be more precise as the present us on the wireframe below :point_down: 

![Screenshot from 2023-04-01 19-28-22](https://user-images.githubusercontent.com/88894525/229307117-169e128d-151e-4d5a-abff-7d55c14818db.png)

BUt this what we can see 
![Screenshot from 2023-04-01 20-02-41](https://user-images.githubusercontent.com/88894525/229307185-8ce10d47-8818-49e1-991f-8b4dc852bf46.png)

_Kindly fix this to be more specific on time and users can't be confused about the time the food will be ready :wink: _